### ISSUE TITLE
HTML tables are no more rendered in PPTX output 

### ISSUE THREAD

**Issue by cderv** [quarto-dev/quarto-cli#9365]
### Discussed in https://github.com/quarto-dev/quarto-cli/discussions/9363

<div type='discussions-op-text'>

<sup>Originally posted by **andresrcs** April 13, 2024</sup>
### Description

I used to be able to insert `gt` tables in PowerPoint presentations seamlessly but It has stopped working, here you can find a minimal reproducible example. If you render this code, it produces a presentation with no table in it.

I'm using `quarto` version 1.4.553 that comes bundled with RStudio (daily build 2024.04.0 Build 693)

````qmd
---
title: "Test"
format: pptx
editor: visual
---

```{r setup}
library(gt)
```

## Test Table

```{r}
#|label: test

iris %>% 
    gt() %>%
    as_raw_html()

```
````</div>

**Comment by cderv** [comment]
This works in 1.3.450 but no more in 1.4.553 nor 1.5

````markdown
---
title: "Tables"
format: 
  pptx: default
keep-md: true
---

# Test table

```{=html}
<table>
<thead>
  <tr>
    <th>Sepal.Length</th>
    <th>Sepal.Width</th>
  </tr>
</thead>
<tbody>
<tr>
  <td>5.1</td>
  <td>3.5</td>
</tr>
<tr>
  <td>4.9</td>
  <td>3.0</td>
</tr>
</tbody>
</table>
```

````

We document HTML tables are a way to create table for all output formats 
https://quarto.org/docs/authoring/tables.html#html-tables

I believe this is a regression

**Comment by cscheid** [comment]
I wonder if what's going on here is that the PPTX writer only supports Pandoc's [SimpleTable](https://pandoc.org/lua-filters.html#type-simpletable), and the tables that are coming from the HTML reader are the fancy kind.

**Comment by andresrcs** [comment]
I have tested the fix using quarto 1.5.36 and it works in the sense that HTML tables are now displayed but they are getting inserted in the top right corner instead of in the "body" of the pptx layout as it was the case before

**Comment by andresrcs** [comment]
Yes, I'm referring to the first example, but I have tested it with a real world pptx and the problem is the same.
